content-src/experiments/send.yaml

```diff
@@ -19,7 +19,7 @@ discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/send'
 privacy_notice_url: 'https://github.com/mozilla/send/blob/master/docs/metrics.md'
 measurements:
   - >
-    When you use Send, Mozilla receives an encrypted copy of the file you upload, and basic information about the file, such as filename and file size. Mozilla does not have the ability to access the content of your encrypted file, and only keeps it for the time or number of downloads indicated.
+    When you use Send, Mozilla receives an encrypted copy of the file you upload, and basic information about the file, such as filename, file hash and file size. Mozilla does not have the ability to access the content of your encrypted file, and only keeps it for the time or number of downloads indicated.
```

Comment:
This needs an [`_l10nsuffix`](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization)